features/support/capybara.rb

```diff
@@ -1,117 +1,84 @@
 # frozen_string_literal: true
 
-require 'selenium/webdriver'
-require 'capybara/minitest'
-include Capybara::Minitest::Assertions
+# require 'selenium/webdriver'
+require 'capybara/rails'
 
-# in case firefox is needed!
-#Capybara.register_driver :selenium do |app|
-#  Capybara::Selenium::Driver.new(app, :browser => :firefox)
-#end
+# Capybara.app = MyRackApp ??
+Capybara.server = :webrick # default is :puma
 
-DEFAULT_JS_DRIVER = :headless_chrome
-# in case firefox is needed!
-# DEFAULT_JS_DRIVER = :headless_firefox
+Capybara.javascript_driver = :selenium_chrome_headless
 
-Capybara.default_driver = :rack_test
-Capybara.javascript_driver = DEFAULT_JS_DRIVER
-Capybara.default_selector    = :css
-Capybara.disable_animation = true
+# include Capybara::Minitest::Assertions
 
-# Capybara 3 changes the default server to Puma. It can be reverted to the previous default of WEBRick by specifying:
-Capybara.server = :webrick
+# DEFAULT_JS_DRIVER = :headless_chrome
+
+# Capybara.default_driver = :rack_test
+# Capybara.javascript_driver = DEFAULT_JS_DRIVER
+# Capybara.default_selector    = :css
+# Capybara.disable_animation = true
 
 # see http://www.elabs.se/blog/60-introducing-capybara-2-1
-Capybara.configure do |config|
-  config.default_driver = :rack_test
-  config.javascript_driver = DEFAULT_JS_DRIVER
-  config.raise_server_errors = true
-  config.match = :prefer_exact
-  config.always_include_port = true
-  config.default_max_wait_time = 10
-end
+# Capybara.configure do |config|
+#   config.default_driver = :rack_test
+#   config.javascript_driver = DEFAULT_JS_DRIVER
+#   config.raise_server_errors = true
+#   config.match = :prefer_exact
+#   config.always_include_port = true
+#   config.default_max_wait_time = 10
+# end
 
 # Needed because cucumber-rails requires capybara/cucumber
 # https://github.com/cucumber/cucumber-rails/blob/7b47bf1dda3368247bf2d45bcb17a224e80ec6fd/lib/cucumber/rails/capybara.rb#L3
 # https://github.com/teamcapybara/capybara/blob/2.18.0/lib/capybara/cucumber.rb#L17-L19
-Before '@javascript' do
-  Capybara.current_driver = DEFAULT_JS_DRIVER
-end
```

Comment:
I remember this `@javascript` tag was heavily patched by us. I had some updates to upstream `capybara/cucumber` but IIRC it required Ruby 3. So I can check about this now we have Ruby 3.1 merged.